examples/text.rs

```diff
@@ -1,6 +1,7 @@
 use std::net::SocketAddr;
 
 use valence::prelude::*;
+use valence_protocol::translation_key;
```

Comment:
Should this be included in the prelude?

Reply:
I think so.

Reply:
It was already in the prelude, but forgot to remove the import from the example.